x-pack/platform/plugins/shared/rule_registry/server/search_strategy/search_strategy.ts

```diff
@@ -139,6 +139,20 @@ export const ruleRegistrySearchStrategyProvider = (
             filter.push(ruleTypeFilter);
           }
 
+          const shouldIncludeDelayedAlerts = request.includeDelayedAlerts ?? false;
+
+          if (!shouldIncludeDelayedAlerts) {
+            filter.push({
+              bool: {
+                must_not: {
+                  term: {
+                    [ALERT_STATUS]: ALERT_STATUS_DELAYED,
+                  },
+                },
+              },
+            });
+          }
+
           const sort = request.sort ?? [];
```

Comment:
This strategy is used by the UI components. We filter out the delayed alerts here so they cannot be listed on the alerts pages.